File: Kate/Programmers/Java/[level 1]/12948_핸드폰 번호 가리기/12948_핸드폰 번호 가리기.js
function solution(phone_number) {
    var answer = '';
    // for문으로 마지막 1~4까지 수 제외한 숫자 *로 replace
    for (let i=0; i < phone_number.length; i++)
        {
            if (i < phone_number.length -4 ) {
                answer += '*'
        }
            else {
                answer += phone_number[i]
            }
        }
    return answer;
}

File: Kate/Programmers/Java/[level 1]/12948_핸드폰 번호 가리기/12948_핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        
        for (int i=0; i < phone_number.length(); i++)
        {
            if (i < phone_number.length()-4 ) {
                answer += "*";
        }
            else {
                answer += phone_number.charAt(i);
                // charAt : 문자열에서 특정 위치의 문자(char)를 리턴
            }
        }
        return answer;
    }
}
